test/browser/features/HappyPath/BAVCheckYourDetailsPage.feature

```diff
@@ -0,0 +1,22 @@
+@success
+Feature: Check Details are Correct
+
+
+  Background:
+    Given a user has navigated to the BAV Landing Page
+    When the user clicks on Continue button
+    Then the user is directed to the Account Details screen
+    Given the user has entered a Sort Code of "12-34-56"
+    Given the user has entered an Account Number of "319268"
+    When the user clicks the Continue button
+    Then the user is directed to the Check Your Answers screen
+
+  Scenario: User routed to loading Bank Details Check Screen
+    When they click on the Continue to bank details check button
+    Then the Loading Bank Details check Screen is served by the FE
```

Comment:
Can we keep the BDD steps consistent please? 

e.g. the user is directed to the Loading Bank Details screen